src/EnergyPlus/FluidProperties.hh

```diff
@@ -60,6 +60,7 @@
 #include <EnergyPlus/Data/BaseData.hh>
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/Psychrometrics.hh>
```

Comment:
I'm guessing this was included to get the warning macros?  I wonder if we should just define those in a separate file rather than having to include the whole psychrometrics header in here.  Maybe I'm missing something though.

Reply:
Correct. I guess if we need to silence other warnings we can move it to a standalone file.